shinysdr/telemetry.py

```diff
@@ -215,8 +215,9 @@ def __maybe_schedule_flush(self):
         
         if self.__expiry_times:
             next_expiry = min(self.__expiry_times.itervalues())
+            until_expiry = max(0, next_expiry - self.__time_source.seconds())
```

Comment:
I find the name `until_expiry` a bit cryptic, especially in context. `interval_until_expiry` maybe?